src/pyFAI/geometryRefinement.py

```diff
@@ -71,6 +71,23 @@
 
 
 class GeometryRefinement(AzimuthalIntegrator):
+    _IMMUTABLE_ATTRS = AzimuthalIntegrator._IMMUTABLE_ATTRS + (
```

Comment:
Probably all float in the tuple.
Is it given that the attributes exist always?

Reply:
Those are all declared and are immutable objects, so can be simply copied, no extra care needed.